web/src/js/components/proxyapp.js

```diff
@@ -55,6 +55,8 @@ var ProxyAppMain = React.createClass({
         };
     },
     focus: function () {
+        document.activeElement.blur();
+        window.getSelection().removeAllRanges();
```

Comment:
@mhils They comes from `ChildFocus.returnFocus`. Not sure if  `activeElement.blur()` is necessary. But for convenience I just moved everything here.

See https://github.com/mitmproxy/mitmproxy/blob/master/web/src/js/components/common.js#L59